apps/backend/src/users/users.service.ts

```diff
@@ -41,4 +35,37 @@ export class UsersService {
 
     return users;
   }
+
+  async findOne(userId: number) {
+    const user = await this.usersRepository.findOneBy({ id: userId });
```

Comment:
❓ All documents created in MongoDB automatically have an `_id` property attached to them of type `ObjectID.` Is `userId` an additional field that all documents are expected to have? It seems like this expects all documents to have an `id` property in addition to the default `_id.` Is this the expected behavior or are we expecting all documents to have a `userId` and `_id` property? If so, the following change should probably be made `await this.usersRepository.findOneBy({ userId };`. I mention this because the `getCurrentMember()` function returns a document with a `userId` field and not an `id` field.